Yesod/Auth/OAuth2/Github.hs

```diff
@@ -47,12 +49,14 @@ instance FromJSON GithubUser where
     parseJSON _ = mzero
 
 data GithubUserEmail = GithubUserEmail
-    { githubUserEmail :: Text
+    { githubUserEmailEmail :: Text
```

Comment:
fair enough; thinking about it, it'd pretty much what is there but the GithubUser would have a field `emails :: [GithubUserEmail]`, and the `GithubUserEmail` would be exactly what's there and probably instead of `..EmailEmail` it'd be better as `...EmailAddress`.

so yeah, i agree `EmailAddress` is better than what i have. happy to change it